openeo/local/connection.py

```diff
@@ -253,6 +253,15 @@ def load_stac(
         cube.metadata = metadata
         return cube
 
+    def list_udf_runtimes(self) -> dict:
+        """
+        Loads all available UDF runtimes.
+
+        :return: data_dict: Dict All available UDF runtimes
+        """
+        runtimes = {"Python": "Python"}
```

Comment:
see https://openeo.org/documentation/1.0/developers/api/reference.html#tag/Capabilities/operation/list-udf-runtimes for expected schema here. e.g. should be something like

```json
{
  "Python": {
    "title": "Python 3",
    "type": "language",
    "versions": {"3": {"libraries": {}}},
    "default": "3"
  }
}
```

Reply:
FYI: also see https://openeo.vito.be/openeo/1.1/udf_runtimes for what VITO backend currently returns in practice